working dir/widgets/inspector/Inspector.kt

```diff
@@ -148,10 +157,22 @@ class Inspector(widget: Widget<*>): SimpleController(widget), FileExplorerFeatur
         fileRoot.findAnyChild(file, File::isAnyChildOf)?.expandToRootAndSelect(tree)
     }
 
+    @Suppress("UNCHECKED_CAST")
     override fun open(data: Any?) {
-        val item = tree(data ?: "<none>")
-        tree.root.children += item
-        tree.expandToRootAndSelect(item)
+        when {
+            data is Window -> exploreWindow(data)
+            data is File -> exploreFile(data)
+            data is Node -> exploreNode(data)
+            (data is Collection<*> && data.getElementType()==File::class.java) -> {
+                println("no way")
```

Comment:
what?

Reply:
Oh man!

Reply:
Fixed. Forgotten debug statement from when I was going crazy over weird behavior:

```
(data is Collection<*> && data.getElementType()==File::class.java) -> {...}
data is Collection<*> && data.getElementType()==File::class.java -> {...}
```
For some reason the two statements are not equivalent, the second will not invoke the block even if 'it is true'. Why are parenthesis in when matcher necessary?